modules/packages/UnitTest.chpl

```diff
@@ -646,8 +646,8 @@ module UnitTest {
       }
       if tmpString == "" {
         for i in 1..len1 {
-          var item1 = seq1[i],
-              item2 = seq2[i];
+          var item1 = seq1[i-isString(seq1)],
+              item2 = seq2[i-isString(seq1)];
```

Comment:
This (and a similar instance below) is probably not what we want here. But I am not well-versed about the UnitTest module to suggest something better. Either the loop indices should change to have 0-based iteration, or a check to see whether something is string or bytes (not just string) should be added. As awkward as it sounds, this also got me thinking whether it'd make sense to have `string.domain` and `bytes.domain` to support generic programming for array-like values.

Reply:
I think @ben-albrecht happily fixed this since this commit and comment.